tests-integration/Elastic.Assembler.IntegrationTests/AssembleFixture.cs

```diff
@@ -27,7 +28,12 @@ public static TBuilder WithEmptyParameters<TBuilder>(this TBuilder builder)
 		foreach (var parameter in parameters)
 			builder.Configuration[$"Parameters:{parameter.Name}"] = string.Empty;
 
-		builder.Configuration[$"Parameters:DocumentationElasticUrl"] = "http://localhost.example:9200";
+		var configBuilder = new ConfigurationBuilder();
+		_ = configBuilder.AddUserSecrets("72f50f33-6fb9-4d08-bff3-39568fe370b3");
```

Comment:
Q: I can see this string 4 times across files. Is this just coincidence or should we put this into a constant?